mapie/classification.py

```diff
@@ -292,9 +303,18 @@ def _check_cv(
         """
         if cv is None:
             return "prefit"
```

Comment:
Can we align this part with `MapieRegressor` and have a `KFold` by default ? if so, need a docstring update in `MapieClassifier`

Reply:
Mmh, do we want to use cross-conformal method as default also for the classification task ? It seems to me that split-conformal is more used than cross-conformal for classification, especially in light with the heavy models used for image classification.

Reply:
At Quantmetry at least, most multi-class problems are not image problems. I would privilege consistency across classes to reduce complexity.

Reply:
OK, `MapieRegressor` and `MapieClassifier` use the same `check_cv` function moved in `utils.py`. `cv=None` calls `KFold(n_splits=5)` for both classes.